lib/encryption/index.js

```diff
@@ -0,0 +1,7 @@
+const { encryptString } = require('./encrypt')
+const { isMatch } = require('./compare')
+
+module.exports = {
+  encryptString,
```

Comment:
As `index` acts as the access point to the module, we can expose what we want from each file.  This is a simple example but you could have complex node modules with many files requiring each other.  This top level exposes only what consumers need